src/main/resources/license-data.json

```diff
@@ -2062,6 +2106,94 @@
       "license": "Apache-2.0",
       "licenseSource": "data"
     },
+    "org.codehaus.groovy:groovy-ant": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-cli-commons": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-groovysh": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-console": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-groovydoc": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-docgenerator": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-cli-picocli": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-datetime": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-jmx": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-json": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-jsr223": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-macro": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-nio": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-servlet": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-sql": {
+      "license": "Apache-2.0",
+      "licenseSource": "data"
+    },
+    "org.codehaus.groovy:groovy-swing": {
```

Comment:
Where do we use it?

Reply:
It's a transient dependency of groovy, just like `bsh`, `jcommander` and `picocli` are of some other library, but the license check plugin complains about them.